cromulent/multiple_instantiation.py

```diff
@@ -1,30 +1,43 @@
 
-# This assumes the default CIDOC-CRM, even though the model code 
+# This assumes the default CIDOC-CRM, even though the model code
 # can generate classes for any ontology
 
 import inspect
-from .model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject	
+
+# gvp 'from model' instead 'from .model'
+from model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject
+#from .model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject
```

Comment:
This is an intra-package reference to ensure that it finds the correct 'model'. See:
   * https://docs.python.org/2/tutorial/modules.html#intra-package-references
   * https://docs.python.org/3/tutorial/modules.html#intra-package-references

Does .model not work in windows? (it should!)